test/testenv/appframework_utils.go

```diff
@@ -12,3 +16,22 @@ func GenerateAppSourceSpec(appSourceName string, appSourceLocation string, appSo
 		AppSourceDefaultSpec: appSourceDefaultSpec,
 	}
 }
+
+// GetAppStatus Get the app install status and enabled status
+func GetAppStatus(deployment *Deployment, podName string, ns string, appname string) (string, string, error) {
+	filePath := fmt.Sprintf("/opt/splunk/etc/apps/%s/default/app.conf", appname)
```

Comment:
What if there is a `local/app.conf` that disables the app?  For ansible in the clustered cases the app is installed locally on the CM/Deployer, copied to the `manager-apps`/`shcluster`, pushed to the cluster, then disabled locally via `local/app.conf`:
```
[splunk@splunk-shc1-deployer-0 splunk]$ cat etc/apps/application1/local/app.conf 
[ui]

[launcher]

[install]
state = disabled
```

Reply:
Updated. Using splunk command and btool to check for app status